framework/resources/Functional/filter/selectivity/negative/q1.res

```diff
@@ -0,0 +1 @@
+.*VALIDATION ERROR: Option planner.filter.min_selectivity_estimate_factor must be between 0.000000 and 1.000000.
```

Comment:
Can you refer to expected files in other negative tests - there seems to be an error code on the first line.